src/test/java/stepdefs/FiltersPageStepDefs.java

```diff
@@ -0,0 +1,45 @@
+package stepdefs;
+
+import buisness.FiltersPage;
+import buisness.LauncherPage;
+import buisness.components.AddFilterComponent;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Then;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.List;
+
+@Slf4j
+public class FiltersPageStepDefs {
+  FiltersPage filtersPage = new FiltersPage();
+  LauncherPage launcherPage = new LauncherPage();
+  AddFilterComponent addFilterComponent = new AddFilterComponent();
+
+  @Then("Filters grid contains such filters")
+  public void filterGridContainsSuchFilters(final DataTable dataTable) {
+    filtersPage.filtersPageShouldBeOpened();
+    filtersPage.gridShouldContainsFilters();
+    List<String> filterNames = dataTable.asList();
+    filterNames.forEach(e -> filtersPage.filterByNameShouldBeVisible(e));
```

Comment:
from my point of view using selenide for assertion is a bad practice. I suggest to use JUinit/TestNG/AssertJ for assertion purposes with clear error message in case of test failure